share/docker/openfast_ubuntu/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenFAST docker images
```

Comment:
I wonder if we should put this into the `docs` (or link it into there) so that it shows up on readthedocs?

On the other hand, it is really nice to have right here with the dockerfiles.

Reply:
I think it's good to have it here next to the dockerfile so it's harder to forget to update when changes are made. I think linking from readthedocs is a good idea too as we only have to update it in one place.

Reply:
I think linking to the README.md file from the docs is the way to go.

Reply:
Docs updated and readme linked